Notes/NotesListViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  NotesListViewModel.swift
+//  Notes
+//
+//  Created by Lena Vadakkel on 10.11.23.
+//
+
+import Foundation
+
+final class NotesListViewModel: ObservableObject {
+    @Published var allNotesFromUser: [Note] = []
+    let dateFormatter: DateFormatter
+	//let presistenceController = PersistenceController()
+	
+    init() {
+        dateFormatter = DateFormatter()
+        dateFormatter.timeStyle = .short
+        dateFormatter.dateStyle = .short
+    }
+	
+	enum ScreenEvent {
```

Comment:
all enum declaration should be placed at the very beginning of your struct/class.